libs/cloud/odc/aws/queue.py

```diff
@@ -180,3 +184,16 @@ def get_messages(
         raise Exception(f"Limit {limit} is not valid.")
 
     return itertools.islice(messages, limit)
+
+
+def publish_to_topic(action, stac):
+    """
+    Publish a message to an sns topic
+    """
+    sns = boto3.resource("sns")
+    sns_topic = sns.create_topic(Name=SNS_TOPIC_NAME)
+
+    stac = json.dumps(stac)
+    sns_topic.publish(
+        Message='{"action": {action}, "stac": {stac}}',
```

Comment:
I think you probably want

```
message_attributes = {
  "action": action,
   ... # more attributes? e.g., product_name?
}
sns_topic.publish(Message=stac, MessageAttributes=message_attributes)
```

Reply:
prob enough to use [SNS client](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sns.html#SNS.Client.publish)